scripts/run_lighthouse_tests.py

```diff
@@ -0,0 +1,173 @@
+            go_file_name, extension)
+
+        python_utils.url_retrieve(url_to_retrieve, outfile_name)
+
+        subprocess.check_call([
+            'tar', '-xvzf', outfile_name, '-C', common.OPPIA_TOOLS_DIR
+        ])
+
+    os.rename(
+        os.path.join(common.OPPIA_TOOLS_DIR, 'go'),
+        GO_PATH)
+    python_utils.PRINT('Installation successful')
+
+
+def download_and_install_compy():
+    """Download and install the compy proxy server."""
+    subprocess.check_call([GO_BINARY, 'get', 'github.com/barnacs/compy'])
+    with common.CD(COMPY_DOWNLOAD_PATH):
+        subprocess.check_call([GO_BINARY, 'install'])
+    python_utils.PRINT('Installation successful')
+
+
+def cleanup():
+    """Kill the running subprocesses and server fired in this program."""
+    dev_appserver_path = '%s/dev_appserver.py' % common.GOOGLE_APP_ENGINE_HOME
+    processes_to_kill = [
+        '.*%s.*' % re.escape(dev_appserver_path),
+        '.*%s.*' % re.escape(COMPY_BINARY)
+    ]
+
+    for p in RUNNING_PROCESSES:
+        p.kill()
+
+    for p in processes_to_kill:
+        common.kill_processes_based_on_regex(p)
+
+
+def delete_reports():
+    """manually delete lighthouse reports to handle
+    lighthouse ci bug where html reports aren't deleted.
+    """
+    shutil.rmtree('.lighthouseci')
+
+
+def start_proxy_server():
+    """Start compy proxy server to serve gzipped assets."""
+    ssl_options = '-cert cert.crt -key cert.key -ca ca.crt -cakey ca.key'
+    p = subprocess.Popen([COMPY_BINARY, '-host', ':%s' % COMPY_SERVER_PORT,
+                          '-gzip', '9', ssl_options])
+    RUNNING_PROCESSES.append(p)
+
+
+def wait_for_port_to_be_open(port_number):
+    """Wait until the port is open.
+    Args:
+        port_number: int. The port number to wait.
+    """
+    waited_seconds = 0
+    while (not common.is_port_open(port_number) and
+           waited_seconds < MAX_WAIT_TIME_FOR_PORT_TO_OPEN_SECS):
+        time.sleep(10)
+        waited_seconds += 2
+    if (waited_seconds ==
+            MAX_WAIT_TIME_FOR_PORT_TO_OPEN_SECS and
+            not common.is_port_open(port_number)):
+        python_utils.PRINT(
+            'Failed to start server on port %s, exiting ...' % port_number)
+        sys.exit(1)
+
+
+def run_lighthouse_checks():
+    """Runs the lighthouserc.js config with bash command lhci autorun."""
+    node_path = os.path.join(common.NODE_PATH, 'bin', 'node')
+    lhci_path = os.path.join(
+        'node_modules', '@lhci', 'cli', 'src', 'cli.js')
+    bash_command = [node_path, lhci_path, 'autorun']
+    python_utils.PRINT(bash_command, 'lhci autorun')
+    process = subprocess.Popen(bash_command, stdout=subprocess.PIPE)
+
+    for line in iter(process.stdout.readline, ''):
+        python_utils.PRINT(line[:-1])
+
+
+def main():
+    """Runs lighthouse checks and deletes reports."""
+    # setup_and_install_dependencies()
+    run_lighthouse_checks()
+    delete_reports()
+
+
+if __name__ == '__main__':
+    main()
```

Comment:
Can you please remove all the compy related code from this file? I don't think we will be using it.

Reply:
Done